tests/element_handle_test.go

```diff
@@ -279,6 +299,9 @@ func TestElementHandleIsChecked(t *testing.T) {
 }
 
 func TestElementHandleQueryAll(t *testing.T) {
+	// tests are faster than parallel execution.
+	// do not make this test parallel.
```

Comment:
Can you clarify this please? This does not seem correct to me :thinking: 
Also, even if we want subtests to be sequential, should at least `TestElementHandleQueryAll` be parallel?

Reply:
Nice catch! I forgot why I jot it down there like that. I'll make this parallel, including subtests.

Reply:
See: [9cd296e](https://github.com/grafana/xk6-browser/pull/1019/commits/9cd296ea60f7902d46146703d1b8248b0cccc233)